### ISSUE TITLE
Mobile Menu: Fellowship Site

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#1084]
## Context
When this page was designed, this was the expected behavior ("Menu" word would show up in the Nav and "Fellowships would be available to be clicked) ":
![mobile-menu-mockup](https://user-images.githubusercontent.com/22157921/36515215-1125e66e-172d-11e8-86b5-fbf32e769434.gif)

The current live page shows the "Fellowships" highlighted instead of "Menu", which can be confusing for users since the highlighted word can be clicked and it will take users to a new page.
![mobile-menu-live-site](https://user-images.githubusercontent.com/22157921/36515278-5f1d32c8-172d-11e8-9c52-fa37502ac6c9.gif)

## Solution
My suggestion would be to implement as designed. @xmatthewx if my memory serves me correctly, we had conversations about the menu behavior and we agreed on the designed version (let me know if I remembered this wrong). If we are going with the designed version, how complicated you think it is to update the current version, @mmmavis . Too much trouble? I would love to get your expertise on that!

**Comment by xmatthewx** [comment]
@taisdesouzalessa - where is the issue where we tackled this recently for cms authored pages? We should align the interaction patterns. 

@mmmavis - hold off on making any changes just yet.

**Comment by taisdesouzalessa** [comment]
@xmatthewx I think you may be referring to this issue here? https://github.com/mozilla/foundation.mozilla.org/issues/977 - let me know if it is another one and I'll try to find it :)

If the assumption above is true, that issue was related to displaying the item on the top of the menu once users tap it (which happens on this model from this ticket). In that ticket, we solved the mobile view for this desktop scenario:

<img width="750" alt="image" src="https://user-images.githubusercontent.com/22157921/36554081-f82f4282-17b2-11e8-8266-3a5167e62f64.png">


------
This ticket is to fix what will be displayed in the mobile menu in a breadcrumb desktop scenario, in which no top nav item is highlighted.

<img width="656" alt="image" src="https://user-images.githubusercontent.com/22157921/36553959-9db75e02-17b2-11e8-8901-30a29011c453.png">

Does that make sense? This is a confusing issue...lol.

**Comment by xmatthewx** [comment]
I suggest that we focus on removing the underline the parent page when on a subpage, and not worry about the display of Fellowship instead of Menu. Fellowship is the site section, so it feels okay and not crazy to me.

**Comment by mmmavis** [comment]
Is this ready for implementation? @taisdesouzalessa @xmatthewx

**Comment by taisdesouzalessa** [comment]
@mmmavis - yep we are following @xmatthewx's suggestions from the comment above.

Basically, we have to remove the yellow line on any page who has a breadcrumb. Example:
<img width="175" alt="image" src="https://user-images.githubusercontent.com/22157921/36927420-5e7af81c-1e32-11e8-969e-896d32651485.png">

**Implementation example:**

Mobile:
<img width="798" alt="image" src="https://user-images.githubusercontent.com/22157921/36927340-d185e21e-1e31-11e8-9884-ffdd1d9dbd4a.png">

Desktop:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/22157921/36927403-4597df22-1e32-11e8-82b1-e22b0961b971.png">


This currently happens only on those pages (let me know if I am missing some):
[Directory > Science Fellows](https://foundation-mofostaging--pr-996.herokuapp.com/fellowships/directory/science/)
[Directory > Open Web Fellows](https://foundation-mofostaging--pr-996.herokuapp.com/fellowships/directory/open-web/)
[Directory > Tech Policy Fellows](https://foundation-mofostaging--pr-996.herokuapp.com/fellowships/directory/tech-policy/)
[Directory > Media Fellows](https://foundation-mofostaging--pr-996.herokuapp.com/fellowships/directory/media/)

**Comment by mmmavis** [comment]
> Basically, we have to remove the yellow line on any page who has a breadcrumb.

I guess that applies to all child pages under `/fellowships/directory`? e.g., `/fellowships/directory/<path-name>`?

What happens if user is on the main "Directory" page. That page does NOT have a breadcrumb. Do we wanna keep or remove yellow underline on the "Directory" nav item?

**Comment by taisdesouzalessa** [comment]
> What happens if user is on the main "Directory" page. That page does NOT have a breadcrumb. Do we wanna keep or remove yellow underline on the "Directory" nav item?

The main Directory page would have the yellow underline as in this example here:
<img width="211" alt="image" src="https://user-images.githubusercontent.com/22157921/36990246-eaea956e-2058-11e8-98a2-9533060fb9c2.png">

**Comment by taisdesouzalessa** [comment]
UPDATE: talked to @mmmavis and we decided to keep the line to orient users on this page, even though "Directory" can still be clickable and take users to parent page. 

We are following the same interaction model as seen in MailChimp (https://mailchimp.com/features/marketing-automation/)

If in the future we see users get confused (by either user testing this page or looking into analytics), we can discuss other ways this interaction can be done.

cc: @xmatthewx - so you are aware of the discussion and conclusion :).

Closing this issue.
